content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -659,6 +668,45 @@ there is no automatic Job restart once the Job status is `type: Failed`.
 That is, the Job termination mechanisms activated with `.spec.activeDeadlineSeconds`
 and `.spec.backoffLimit` result in a permanent Job failure that requires manual intervention to resolve.
 
+### Terminal Job conditions
+
+A Job has two possible terminal states, it ends up either succeeded, or failed,
+and these states are reflected by the presence of the Job conditions `Complete`
+or `Failed`, respectively.
+
+The failure scenarios encompass:
+- the `.spec.backoffLimit`
```

Comment:
```suggestion
- the `.spec.backoffLimit` is reached
```

Maybe a full sentence would be better as similar to "the .spec.completions is reached".

Reply:
Yeah, but I think in case of backoffLimit "exceeded" is better. This is also what the reason on the Failed condition says.

Reply:
That sounds more reasonable to me. Thanks!